src/static/css/2019.css

```diff
@@ -1317,10 +1327,13 @@ p.copyright a {
 
 .footer .nav-dropdown-list {
   bottom: 60px;
-  flex-direction: column-reverse;
   border-radius: 29px 29px 29px 0;
 }
 
+.footer:not(.table-of-contents) .nav-dropdown-list {
+  flex-direction: column-reverse;
+}
+
```

Comment:
Yeah that's an intentional reverse. It's a little bit odd but think the best way as it's going up, instead of the usual down.

Will resolve this now. Thanks!